game.libretro.prboom/addon.xml

```diff
@@ -1,32 +1,31 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="game.libretro.prboom"
-		name="PrBoom"
-		version="2.5.0"
-		provider-name="Libretro">
-	<requires>
-		<import addon="game.libretro" version="1.0.0"/>
-	</requires>
-	<extension point="kodi.gameclient"
+<addon id="game.libretro.prboom" name="PrBoom" version="2.5.0" provider-name="Libretro">
+    <requires>
+        <import addon="game.libretro" version="1.0.0"/>
+    </requires>
+    <extension point="kodi.gameclient"
                    library_android="libgame.libretro.prboom.so"
                    library_linux="game.libretro.prboom.so"
                    library_osx="game.libretro.prboom.dylib"
                    library_windx="game.libretro.prboom.dll"
                    library_wingl="game.libretro.prboom.dll">
-		<platforms></platforms>
-		<extensions>wad|iwad</extensions>
-		<supports_vfs>true</supports_vfs>
-	</extension>
-	<extension point="xbmc.addon.metadata">
-		<summary lang="en">PrBoom v2.5.0</summary>
-		<description>PrBoom is a version of the classic 3D shoot'em'up game Doom, originally written by id Software.
-		
+        <platforms></platforms>
+        <extensions>wad|iwad</extensions>
+        <supports_vfs>true</supports_vfs>
+        <supports_game_loop>true</supports_game_loop>
```

Comment:
This property is being dropped from the game API. it doesn't hurt, so you don't need to update the PR, just for future reference